virt/virtual_machines/virtual_disks/virt-managing-offline-vm-snapshots.adoc

```diff
@@ -1,21 +1,44 @@
-[id="virt-managing-offline-vm-snapshots"]
-= Managing offline virtual machine snapshots
+[id="virt-managing-vm-snapshots"]
+= Managing virtual machine snapshots
 include::modules/virt-document-attributes.adoc[]
-:context: virt-managing-offline-vm-snapshots
+:context: virt-managing-vm-snapshots
 
 toc::[]
 
-You can create, restore, and delete virtual machine (VM) snapshots for VMs that are powered off (offline). {VirtProductName} supports
-offline VM snapshots on:
+You can create, restore, and delete virtual machine (VM) snapshots for VMs, whether the VMs are powered off (offline) or on (online). {VirtProductName} supports offline VM snapshots on:
 
 * Red Hat OpenShift Container Storage
 * Any other storage provider with the Container Storage Interface (CSI) driver that supports the Kubernetes Volume Snapshot API
 
+[NOTE]
+====
+Online snapshots are not supported for hot-plugged virtual disks.
+====
+
+[IMPORTANT]
+====
+In order to create snapshots of an online (Running state) VM, you must install the QEMU guest agent.
```

Comment:
Here too

Reply:
This one is the old version (OFFLINE in file name). The new assembly contains this:

[NOTE]
====
In order to create snapshots of an online (Running state) VM with the highest integrity, we recommend you install the QEMU guest agent.

The QEMU guest agent ensures a consistent snapshot by attempting to quiesce the VM's filesystem. This insures that in-flight I/O is written to the disk before the snapshot is taken and ensures that VMs with multiple disks have a coherent snapshot. If the guest agent is not present, quiescing is not possible and a best effort snapshot is taken. The conditions under which the snapshot was taken are reflected in the snapshot indications that are displayed in the web console or CLI.
====